std/numeric/package.d

```diff
@@ -1647,45 +1653,6 @@ unittest
 }
 
 /**
-Computes accurate sum of binary logarithms of input range $(D r).
- */
-ElementType!Range sumOfLog2s(Range)(Range r)
```

Comment:
It will be released in `2.067`. An alias is not needed since `std.numeric.summation` is publicly imported in `std.numeric`.